uint8 can_id
uint8 frame_type
uint8 channel
uint8 data_length
uint8[8] data

